hnn_core/tests/test_cell_response.py

```diff
@@ -24,7 +24,11 @@ def test_cell_response(tmp_path):
                                  spike_gids=spike_gids,
                                  spike_types=spike_types,
                                  times=sim_times)
-    cell_response.plot_spikes_hist(show=False)
+    kwargs_hist = dict(alpha=0.25)
+    fig = cell_response.plot_spikes_hist(show=False, **kwargs_hist)
+    assert all(patch.get_alpha() == kwargs_hist['alpha']
+           for patch in fig.axes[0].patches), \
```

Comment:
yeah this is what I was looking for!

pep8 asks to avoid backslash for stylistic reasons: https://peps.python.org/pep-0008/

do you have a linter on your editor? You can use the "Python" package on VS Code ... it usually comes with a linter. You will also need pep8 installed in your python environment